src/ts/components/playbacktogglebutton.ts

```diff
@@ -48,6 +63,7 @@ export class PlaybackToggleButton extends ToggleButton<ToggleButtonConfig> {
     // Call handler upon these events
     player.on(player.exports.PlayerEvent.Play, (e) => {
       this.isPlayInitiated = true;
```

Comment:
could we re-use this flag as opposed to introduce a new one?

Reply:
No we can't. This flag is e.g. reset on `Paused` or `Playing`, which would lead to undesired side effects. For example, the user exits fullscreen, pauses the player (the flag is reset) and when resuming again, fullscreen would be triggered again.

Reply:
👍